### DISCUSSION TITLE
VIC20 - available games

### DISCUSSION THREAD

**Discussion by tusigt** [fdivitto/FabGL#172] [Q&A]
Hi!

Is there a list of games, that work under the VIC20 example sketch?

Tnx

**Comment by fdivitto** [comment]
The list of tested games is the download list, at:
[http://cloud.cbm8bit.com/adamcost/vic20list.txt](http://cloud.cbm8bit.com/adamcost/vic20list.txt)

**Comment by gnokketto** [comment]
the site non working, and the related function to emulator "LIST" not download, its possible to fix it to add other games to emulator ?
thanks
